consensus/e2e_tests/utils_test.go

```diff
@@ -153,12 +155,19 @@ func GenerateBuses(t *testing.T, runtimeMgrs []*runtime.Manager) (buses []module
 }
 
 // CLEANUP: Reduce package scope visibility in the consensus test module
-func StartAllTestPocketNodes(t *testing.T, pocketNodes IdToNodeMapping) {
+func StartAllTestPocketNodes(t *testing.T, pocketNodes IdToNodeMapping) error {
 	for _, pocketNode := range pocketNodes {
 		go startNode(t, pocketNode)
 		startEvent := pocketNode.GetBus().GetBusEvent()
 		require.Equal(t, startEvent.GetContentType(), messaging.NodeStartedEventType)
+		if err := pocketNode.GetBus().GetStateMachineModule().SendEvent(coreTypes.StateMachineEvent_Start); err != nil {
```

Comment:
You can create a local `stateMachine := pocketNode.GetBus().GetStateMachineModule()` so this is easier to read